prdoc/pr_4586.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Removed `pallet::getter` usage from pallet-identity
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removed the `pallet::getter`s from `pallet-identity`.
+      The syntax `StorageItem::<T, I>::get()` should be used instead.
+
+crates:
+  - name: pallet-identity
+    bump: major
```

Comment:
It also needs to mention pallet-alliance with a patch, since it was modified.

Reply:
@xlc getting conflicting comments here. @ggwpez In a comment above was told to remove the semver bumps since they don't change in a 'semver' sense.

I'd say I agree with xlc since there's really no functionality added/bugs fixed, but I'm happy to do whatever!

Reply:
it is not modified. the tests are not published

Reply:
Hadn't seen there was a none option. Have set the bumps to that.